README.md

```diff
@@ -253,6 +253,28 @@ const MyComponent = ({clientId}) => {
 | `slidesPerPage` | `integer` | `1` | Number of slides to show per page |
 | `parentBlockId` | `string` | `''` | Client ID of parent block. This is required.  |
 
+## Optional
+A component that takes care of the logic of rendering nodes only when it is selected.
```

Comment:
I'd love for this description to explain what the value prop is for and how it should be used :)